src/const.ts

```diff
@@ -14,3 +14,10 @@ export const ALLOWANCE_FOR_ENABLED_TOKEN = new BN(2).pow(new BN(128)) // 3402823
 // Model constants
 export const FEE_DENOMINATOR = 1000 // Fee is 1/fee_denominator i.e. 1/1000 = 0.1%
 export const BATCH_TIME = 300
+
+export const LEGALDOCUMENT = {
+  CONTACT_ADDRESS: '[INSERT ADDRESS]',
+  POC_URL: '/',
+  TEXT: '',
```

Comment:
`TEXT` is not used anywhere. Did you intent to put the actual text in here?
If not used, I'd suggest to just remove it.

Reply:
:+1: